domains/murilo.json

```
@@ -0,0 +1,11 @@
+{
+    "description": "My Portfolio Website",
+    "repo": "https://github.com/mbezerraa/mbezerraa.github.io",
+    "owner": {
+        "username": "mbezerraa",
+        "email": "murilo_yanq6517@hotmail.com"
+    },
+    "record": {
+        "CNAME": "https://mbezerraa.github.io"
```

CNAME can't contain protocol
```suggestion
        "CNAME": "mbezerraa.github.io"
```